test/integration/test-visibility-states.js

```diff
@@ -18,6 +18,7 @@ import {BaseElement} from '../../src/base-element';
 import {registerElement} from '../../src/custom-element';
 import {viewerForDoc} from '../../src/services';
 import {documentStateFor} from '../../src/service/document-state';
+import {setViewerVisibilityState} from '../../src/service/viewer-impl';
```

Comment:
Hmm. This function still isn't being found, likely because it's restricted. Any ideas, @jridgewell @erwinmombay @camelburrito?